apps/publishers/tests/portal/members/test_accept_publisher_member_invite.py

```
@@ -0,0 +1,45 @@
+            self.captureOnCommitCallbacks(execute=True),
+        ):
+            member, inv, raw = PublisherMemberInvitationService().create_invitation(
+                publisher=self.publisher,
+                email=email,
+                role="staff",
+                invited_by=baker.make(User),
+            )
+        return member, inv, raw
+
+    def test_accept_activates_member_no_auth_required(self):
+        member, inv, raw = self._invite()
```

Ruff (0.15.15)

[warning] Unpacked variable `inv` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)